src/main/java/de/htwberlin/madlib/MadLib.java

```diff
@@ -8,7 +10,25 @@
 
   public String create() {
     // TODO: implement this method
+    Random randomNumber = new Random();
+
+    int randomAdjective = randomNumber.nextInt(ADJECTIVES.length);
+    int randomVerb = randomNumber.nextInt(VERBS.length);
+    int otherRandomVerb = randomNumber.nextInt(VERBS.length);
+    int randomFamousPerson = randomNumber.nextInt(VERBS.length);
+
+    return "Java programming is so " + ADJECTIVES[randomAdjective] + "! It makes me so excited all the time because I love to " + VERBS[randomVerb] + ". Stay hydrated and " + VERBS[otherRandomVerb] +" like you are " + FAMOUS_PERSONS[randomFamousPerson]+"!";
+  }
```

Comment:
Das passt soweit. Das hier ist die Musterlösung:

```java
public String create() {
  String randomAdjective = randomChoice(ADJECTIVES);
  String randomVerb1 = randomChoice(VERBS);
  String randomVerb2 = randomChoice(VERBS);
  String randomFamousPerson = randomChoice(FAMOUS_PERSONS);

  return String.format(
    "Java programming is so %s! It makes me so excited all the time because I love to %s. Stay hydrated and %s like you are %s!",
    randomAdjective,
    randomVerb1,
    randomVerb2,
    randomFamousPerson
  );
}

private String randomChoice(String[] wordList) {
  return wordList[random.nextInt(wordList.length)];
}
```

Mit String.format() kann man ein String ohne die ganzen `" + "` zusammenbauen.